cumulus_library/studies/core/core_templates/condition.sql.jinja

```diff
@@ -74,9 +74,14 @@ SELECT
     tc.recordeddate,
     tc.recordeddate_week AS recorded_week,
     tc.recordeddate_month AS recorded_month,
-    tc.recordeddate_year AS recorded_year
+    tc.recordeddate_year AS recorded_year,
+    t_clinicalstatus_coding.clinicalstatus_row.code AS clinicalstatus_code,
```

Comment:
I hate that allergies and conditions do a complicated codeable concept status field. Just use a static code field like all the other resources. You think you're too good for a static code, Condition? You're not.